js/nodes/processes/load_process_list_pressure.js

```diff
@@ -0,0 +1,54 @@
+
+function AssignScalarVariableToConditionsProcess() {
+    this.addInput("model_part_name","string");
+    this.addInput("interval","process_array");
+
+    this.properties = {
+        "python_module" : "assign_scalar_variable_to_conditions_process",
+        "kratos_module" : "KratosMultiphysics",
+        "Parameters"    : {
+            "model_part_name" : "Structure.COMPUTE_HROM.SurfacePressure3D_Pressure_on_surfaces_Auto4",
```

Comment:
These are the default parameters that the user will get while adding the node, so I will not initialize them with values that are used specifically for the structural models.